### ISSUE TITLE
elm properties which are text nodes crash snabbdom

### ISSUE THREAD

**Issue by wbern** [snabbdom/snabbdom#330]
```
    function emptyNodeAt(elm) {
        var id = elm.id ? '#' + elm.id : '';
        var c = elm.className ? '.' + elm.className.split(' ').join('.') : '';
        return vnode_1.default(api.tagName(elm).toLowerCase() + id + c, {}, [], undefined, elm);
    }
```

![image](https://user-images.githubusercontent.com/25477382/29821347-d994f36a-8cc7-11e7-9c56-4268c6a8ce5f.png)

Is this normal? Perhaps snabbdom should throw a more sane error.

**Comment by staltz** [comment]
If I'm not mistaken, this should only happen if you use snabbdom's top-level `patch` with Text Nodes. Is that your use case?

**Comment by wbern** [comment]
@staltz yes. Not sure if this is sort of "out of scope" for Snabbdom. Not a huge deal in my case, but I wanted to point it out in any case.

**Comment by caridy** [comment]
yeah, this is a very edge case that we should probably not attempt to fix, but maybe we should add a note somewhere in the README or something to make it clear that invoking patch for non-elements is not supported.

**Comment by mightyiam** [comment]
Could someone who understands exactly what is unsupported please kindly take a moment to make a documentation pull request?
